include/igl/opengl/glfw/imgui/ImGuizmoMenu.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+////////////////////////////////////////////////////////////////////////////////
+#include <imgui/imgui.h>
+#include <imgui/imgui_internal.h>
+#include <imguizmo/ImGuizmo.h>
+#include <Eigen/Dense>
+#include <vector>
+////////////////////////////////////////////////////////////////////////////////
+
+namespace ImGuizmo {
```

Comment:
Can you put that in the namespace `igl::opengl::glfw::imgui` like the rest of the folder?